specs/merge/beacon-chain.md

```diff
@@ -148,6 +166,14 @@ def is_transition_block(state: BeaconState, block_body: BeaconBlockBody) -> bool
     return not is_transition_completed(state) and block_body.execution_payload != ExecutionPayload()
 ```
 
+#### `compute_time_at_slot`
+
+```python
+def compute_time_at_slot(state: BeaconState, slot: Slot) -> uint64:
```

Comment:
It's used safely here, but we may want to add a note about under/overflows in the math here, in case the function is going to be used elsewhere.